### ISSUE TITLE
Request for uploading Dependencies

### ISSUE THREAD

**Issue by escortKwon** [turtlebot/turtlebot4_robot#9]
**Describe the the feature you would like**
I had my TB4 lite clean-installed. Because I had some connection problems and in troubleshooting progress, therefore I thought **overwriting the latest TB4 lite image(ver. 0.1.3) on my TB4** is the best way to solve problems.  
After overwriting image, I had completed setting connection between Raspberry Pi 4 and 5GHz AP, between iCreate3 and 2.4GHz AP.  
with "nmap -sP ***.***.***.***" command, I could find IP addresses from both TB4 lite and iCreate3. But there were no what looks like some name or label(Ubuntu or Create-XXXX)  
**I think the inside dependencies from my TB4 has been removed or not installed, so TB4 could not boot properly.**

In summary, I **need some dependencies from 'turtlebot/turtlebot4'.** I could not run turtlebot4_setup commands without those dependencies. Or if there are other ways which is not uploaded on user manual, please let me know.

Thank you!

![Screenshot from 2022-11-03 10-32-59](https://user-images.githubusercontent.com/78137059/199634139-4b34756c-4020-405e-a7b6-59544209dc88.png)

**Comment by roni-kreinin** [comment]
@escortKwon The 0.1.3 image comes with the `turtlebot4_robot` package pre-installed via debian, so installing it from source is not necessary. That said, the source installation steps are outdated, and the `dependencies.repos` file does not exist anymore. I will update the manual.

Additionally, the `robot_upstart` job should already be installed by default. You can check the service status by calling `sudo systemctl status turtlebot4`.

If you wish to re-install it, I recommend following [these instructions](https://turtlebot.github.io/turtlebot4-user-manual/tutorials/multiple_robots.html#rpi4-robot-upstart-job).

**Comment by escortKwon** [comment]
@roni-kreinin First of all, Thank you for letting me know that I don't have to configure dependencies anymore.

And, I have checked the TB4 status by calling `sudo systemctl status turtlebot4` and the result is as follows
![Screenshot from 2022-11-04 08-44-52](https://user-images.githubusercontent.com/78137059/199855789-6e495e59-4972-4d27-8b05-0dc4b798ec07.png)

Also, I have called `nmap -sP ***.***.***.***` again for detecting RB4 and iCreate3 simultaneously. The good news is that I can detect both of them, but **the bad news is that there're no any names by each IP address.** Could I get some any solutions? (BTW, I have clean-installed by overwriting RB4 SD card on my TB4 lite!)

- 192.168.0.81 - iCreate3 Robot
- 192.168.0.82 - RP4 on TB4

![Screenshot from 2022-11-04 08-50-23](https://user-images.githubusercontent.com/78137059/199856192-18c08ce2-7724-46de-b055-e8d33e22ecfe.png)

In summary, I can detect both RP4 and iCreate3 by calling nmap, **but there're no any topic called `/ip`on my Remote PC.** I just installed Ubuntu 20.04.5 & ROS2 Galactic on my Remote PC. **Do I need to configure more packages on my Remote PC?**

Thank you for you efforts!

**Comment by roni-kreinin** [comment]
Are you able to see any ROS2 topics on your PC? And what about on the RPi4?

**Comment by escortKwon** [comment]
Both of them can display running topics on ROS2. **But it seems abnormal on my Remote PC**
![Screenshot from 2022-11-08 15-02-23](https://user-images.githubusercontent.com/78137059/200487355-fa2b7c40-d0bc-4d10-ad3b-b372bee10ea0.png)
![Screenshot from 2022-11-08 15-02-26](https://user-images.githubusercontent.com/78137059/200487359-52d5ce6a-c8b2-4731-be00-6644a6b8aa59.png)
The upper one is captured from Remote PC and the other one is from RPi4

**Comment by roni-kreinin** [comment]
@escortKwon Did you apply the `DontRoute` configuration to CycloneDDS as described [here](https://turtlebot.github.io/turtlebot4-user-manual/overview/quick_start.html#network-configuration)?

**Comment by escortKwon** [comment]
@roni-kreinin  Absolutely, this is my `cyclonedds_pc.xml` for now.
![Screenshot from 2022-11-17 08-35-16](https://user-images.githubusercontent.com/78137059/202317943-edd3ddeb-ceeb-42bd-982d-7a68377ca8ca.png)
```bash
<Tracing>
            <Verbosity>config</Verbosity>
            <OutputFile>stdout</OutputFile>
</Tracing>
```
And this snippet is from CycloneDDS GitHub Issue I've made. ([Configuring ROS2 on Turtlebot4](https://github.com/eclipse-cyclonedds/cyclonedds/issues/1476)) Also, for now, I'm running a single TB4, therefore I dropped `<NetworkInterfaceAddress>wlp0s20f3</NetworkInterfaceAddress>`
Thanks!

**Comment by escortKwon** [comment]
Using FastDDS discovery server, issue has been resolved. Refer to this [link](https://github.com/turtlebot/turtlebot4/issues/98).
